src/test/url-loader/package-url-resolver_test.ts

```diff
@@ -72,7 +72,7 @@ suite('PackageUrlResolver', function() {
           resolvedUrl`http://abc.xyz/foo.html`);
       assert.equal(
           r.resolve(packageRelativeUrl`//abc.xyz/foo.html`),
-          resolvedUrl`file://abc.xyz/foo.html`);
+          resolvedUrl`//abc.xyz/foo.html`);
```

Comment:
The `//foo.com/bar.html` url isn't a resolved URL. I'd prefer that we resolve to `https://foo.com/bar.html` with an option to specify the protocol (via e.g. a server root url option)